integration/stability/bind_mount_linux.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# This test will start a container with a bind mount
+# and set bind propagation, the purpose of this
+# test is to check if the container stops properly
+
+set -e
+
+cidir=$(dirname "$0")
+testname="${0##*/}"
+sysname=$(uname)
+
+[ "$sysname"x == "Linux"x ] || exit 0
+
+source "${cidir}/../../metrics/lib/common.bash"
```

Comment:
If this is just for the `clean_env` function, it would be better to only source `"${cidir}/../../lib/common.bash"`

Reply:
udpate